test/zzz-correct-index-names.sh

```diff
@@ -42,6 +42,9 @@ cleanup() {
   fpod=$( get_running_pod fluentd )
   oc logs $fpod > $ARTIFACT_DIR/$fpod.log 2>&1
 
+  oc exec $fpod -- env | sort > $ARTIFACT_DIR/env_vars.log 2>&1
+  oc exec $fpod -- find /etc/fluent/configs.d/ -exec cat {} \; $ARTIFACT_DIR/fluent_conf.log 2>&1
```

Comment:
add `-type f` to the find command - not sure what will happen when you do `cat /etc/fluent/configs.d/openshift/`